notifications.bs

```diff
@@ -640,6 +682,7 @@ interface Notification : EventTarget {
   readonly attribute NotificationDirection dir;
   readonly attribute DOMString lang;
   readonly attribute DOMString body;
+  readonly attribute USVString navigate;
```

Comment:
This kinda sounds like a boolean name, navigateTo maybe?

Reply:
It's the same term we use for the Navigation API.